mitosheet/src/components/Mito.tsx

```diff
@@ -356,12 +391,13 @@ export const Mito = (props: MitoProps): JSX.Element => {
                     mitoAPI={props.mitoAPI}
                 />
             )
-            case ModalEnum.Feedback: return (
-                <FeedbackModal
-                    usageTriggereeFeedbackID={uiState.currOpenModal.feedbackID}
+            case ModalEnum.DeleteGraphs: return (
+                <DeleteGraphsModal
                     setUIState={setUIState}
                     mitoAPI={props.mitoAPI}
-                    numUsages={userProfile.numUsages}
+                    sheetIndex={uiState.currOpenModal.sheetIndex}
+                    dependantGraphTabNamesAndIDs={uiState.currOpenModal.dependantGraphTabNamesAndIDs}
+                    dfName={sheetDataArray[uiState.currOpenModal.sheetIndex].dfName}
```

Comment:
This might be undefined! Never just index into the sheet data array and then get a property of the result without checking if defined. This causes sheet crashing bugs (and should be something we handle in the type system after we have sheet ids!).